ios_sample_test_app/ios_sample_test_app/AppDelegate.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  AppDelegate.swift
+//  ios_sample_test_app
+//
+//  Created by Brandon Boothe on 3/23/23.
+//
+
+import UIKit
+
+import BranchSDK
+
+@main 
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+    var navigationcontroller: UINavigationController?
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        
+        //See Branch Logs
+        Branch.getInstance().enableLogging()
+        
+        // This version of initSession includes the source UIScene in the callback
+        BranchScene.shared().initSession(launchOptions: launchOptions, registerDeepLinkHandler: { (params, error, scene) in
+            
+            guard let data = params as? [String: AnyObject] else { return }
+            
+            guard let options = data["nav_to"] as? String else { return }
+            switch options {
+                case "color_block_page":
```

Comment:
Maybe a little nit but why not an `if`, if you only have one case?

Reply:
Added the other case for going to the Read Deep Link Page